lib/usermanager.js

```diff
@@ -53,6 +52,21 @@ exports = module.exports = {
     'DuplicateUserError': DuplicateUserError
   },
 
+  /**
+   * checks if a user is permitted the action on the resource
+   *
+   * @param {string} action
+   * @param {object} user
+   * @param {string} resource
+   * @param {callback} next (function (err, isAllowed))
+   */
+
+  hasPermission: function(action, user, resource, next) {
+    permissions = require('./permissions');
```

Comment:
Remove the circular dependency by removing the `require('./usermanager') and replacing calls to `usermanager` with `app.usermanager`.